package.json

```diff
@@ -122,6 +122,7 @@
     "chai": "^4.3.7",
     "chalk": "^4.x",
     "chromedriver": "latest",
+    "clean-jsdoc-theme": "^4.2.17",
```

Comment:
It was surprisingly difficult to find a jsdoc template that had upgraded to jsdoc 4. Most are still designed only for jsdoc 3 and use the taffydb dependency which jsdoc 4 [has said is no longer supported](https://github.com/jsdoc/jsdoc/tree/main/packages/jsdoc-salty#use-salty-in-a-jsdoc-template). 

This theme is arbitrarily chosen from a short list I could find. Dependencies looked minimal and didn't add anything to `npm audit`